File: Daily Challanges/2026-05-May/2026-05-03.js
function getGreeting(time) {
  const [h, m] = time.split(":").map((item) => Number(item));
  if (h >= 5 && h <= 11) return "Good morning";
  if (h >= 12 && h <= 17) return "Good afternoon";
  if (h >= 18 && h <= 21) return "Good evening";
  return "Good night";
}

console.log(getGreeting("06:30"));
console.log(getGreeting("12:00"));
console.log(getGreeting("21:59"));
console.log(getGreeting("00:01"));
console.log(getGreeting("11:30"));

File: Daily Challanges/2026-05-May/2026-05-03.py
def get_greeting(s):
    h, m = map(int, s.split(':'))
    if  5 <= h < 12:
        return "Good morning"
    if 12 <= h < 18:
        return "Good afternoon"
    if 18 <= h < 22:
        return "Good evening"
    return "Good night"

print(get_greeting("06:30"))
print(get_greeting("12:00"))
print(get_greeting("21:59"))
print(get_greeting("00:01"))
print(get_greeting("11:30"))